packages/block-editor/src/components/writing-flow/use-tab-nav.js

```diff
@@ -28,9 +28,6 @@ export default function useTabNav() {
 		[]
 	);
 
-	// Don't allow tabbing to this element in Navigation mode.
```

Comment:
I moved this to the after div itself for better code context/readability. Took me forever to figure out what this did.